src/main/features/response/views/statement-of-means/supported-by-you.njk

```diff
@@ -0,0 +1,57 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioOption, numericInput, textArea, saveAndContinueButton, errorSummary %}
+
+{% set title = 'Money claim' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Do you support anyone else financially?') }}</h1>
+
+      <form method="post" novalidate
+        class="analytics-submit-event-trigger"
+        data-event-action="SoM: Do you support anyone else financially?"
+        data-event-label-from="doYouSupportAnyone">
+
+        {{ csrfProtection(csrf) }}
+
+        {% set error = form.errorFor('doYouSupportAnyone') %}
+        <fieldset id="doYouSupportAnyone[label]" class="form-group {% if error %} form-group-error {% endif %}">
+          <legend>{{ t('This can include adults and people you don’t live with.') }}</legend>
+
+          {% if error %}<span class="error-message">{{ t(error) }}</span>{% endif %}
+
+          {{
+            radioOption(form = form, label = 'Yes', name = 'doYouSupportAnyone', value = 'true', checked = form.valueFor('doYouSupportAnyone') === true)
+          }}
+
+          {% set atLeastOfNumberShouldBeEnteredError = form.errorFor('numberOfPeople') %}
+
+          <div class="form-group panel panel-border-narrow js-hidden {% if atLeastOfNumberShouldBeEnteredError %} form-group-error {% endif %}"
+               id="doYouSupportAnyone-true" aria-hidden="false">
+
+          {% if atLeastOfNumberShouldBeEnteredError %}
+            <p id="numberOfPeople[label]" class="error-message">
+              {{ t(atLeastOfNumberShouldBeEnteredError) }}
+            </p>
+          {% else %}
+            <p>{{ t('Enter number for each age group.') }}</p>
+          {% endif %}
+
+            {{ numericInput('Number of people', 'numberOfPeople[value]', form) }}
```

Comment:
Your could name arg here for readability as well e.g. `(label = 'Number of people...'`

Reply:
ok

Reply:
done